analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -128,12 +131,19 @@ class Pub(
     private val reader = PubCacheReader()
     private val gradleDefinitionFilesForPubDefinitionFiles = mutableMapOf<File, Set<File>>()
 
+    private val pubDependenciesOnly = options[OPTION_PUB_DEPENDENCIES_ONLY].toBoolean()
+
     override fun transformVersion(output: String) = output.removePrefix("Dart SDK version: ").substringBefore(' ')
 
     override fun getVersionRequirement(): RangesList = RangesListFactory.create("[2.10,)")
 
     override fun beforeResolution(definitionFiles: List<File>) {
-        if (gradleFactory != null) {
+        if (pubDependenciesOnly) {
+            logger.info {
+                "Only analyzing Pub dependencies, skipping additional package managers for Flutter packages" +
+                    " (Gradle for Android, CocoaPods for iOS dependencies)."
```

Comment:
Nit: Please move the continuing space between these lines from the second to the first line, i.e. end the first line with `packages " +` and start the second line with `"(Gradle`.

Reply:
Okay, changed as suggested.